### PR TITLE
Create github action to assign RUSTSEC ids

### PR DESCRIPTION
**Pull request by alex**
Untested, and won't work until the rustec-admin release happens and we bump the versions in this. But it's got all the right pieces I think.

### CODE REVIEW

**Comment by alex** [comment]
@tarcieri can you do a rustsec-admin release so we can move forward with this?

**Comment by tarcieri** [comment]
I'm taking short break from the project but I'll do it when I come back

**Comment by tarcieri** [comment]
`rustsec-admin` v0.2.0 is out and I bumped this repo to use it in #321

**Comment by alex** [comment]
@tarcieri Cool! Updated! Not sure if there's a great way to test this besides to merge it, then merge something and see if it works.

File: .github/workflows/assign-ids.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        fi
+    - name: Assign IDs
+      run: rustsec-admin assign-id
+    - name: Create pull request
+      uses: peter-evans/create-pull-request@v2
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        commit-message: Assign RUSTSEC IDs
+        title: Assign RUSTSEC IDs
+        branch: assign-ids
```
Comment (tarcieri) [inline .github/workflows/assign-ids.yml:33]: It'd be nice if we could get these parameterized around the assigned ID somehow

Comment (alex) [reply .github/workflows/assign-ids.yml:33]: Dos `rustsec-admin assign-id` emit that in any way? Can you think of another good way to get that info?

Comment (tarcieri) [reply .github/workflows/assign-ids.yml:33]: It prints it to standard out, although with some additional text. We could potentially add a mode/flag that just prints the ID on success.

Comment (tarcieri) [reply .github/workflows/assign-ids.yml:33]: (We could also merge this to see if it works and circle back on that particular fixup)

Comment (alex) [reply .github/workflows/assign-ids.yml:33]: I'd be ok with merging and circling back with the improvement :-)

Comment (tarcieri) [reply .github/workflows/assign-ids.yml:33]: Cool. Can you flag this PR as ready-to-review?

Comment (alex) [reply .github/workflows/assign-ids.yml:33]: Done!

**Comment by tarcieri** [comment]
Let's give it a try

**Comment by alex** [comment]
🎉

**Comment by tarcieri** [comment]
Hrmm, well it failed... I assume because it's attempting to open a PR with no changes to the branch?

https://github.com/RustSec/advisory-db/actions/runs/152014712

**Pull request merged**